pkgs/servers/unstructured-api/default.nix

```diff
@@ -13,20 +13,140 @@ let
     packages:
     with packages;
     [
-      unstructured-api-tools
-      unstructured
-      pydantic
+      aiofiles
```

Comment:
Why does this repeat all of the dependencies of `unstructured`?

Reply:
If i remember correctly, i just took the dependencies of upstream as is at that time. It could be that i made a mistake.